plugins/inputs/procstat/win_service_windows.go

```diff
@@ -39,6 +39,7 @@ func queryPidWithWinServiceName(winServiceName string) (uint32, error) {
 	}
 
 	buf = make([]byte, bytesNeeded)
+	//nolint:gosec // G103: Use of unsafe calls should be audited -> call audited
```

Comment:
hmm saying "call audited" is true today, but there is nothing stopping someone from adding something to this or the functions below where this is ignored as well that might be unaudited, right?

Reply:
Very good point, I was having similar doubts as well. Especially for the ones that  do a `//nolint` for a complete function.

Reply:
@powersj 
Your comment is about my `//nolint:gosec` comment which filters issue out only for the nearest line/call. If someone adds another `unsafe.*` call to this line, it will be also filtered out by this `//nolint` - we cannot to much about that.

But I can change all `//nolint:gesec` which cover whole functions to "single-calls" filtering if you want.

Reply:
What do you all think about removing the "-> call audited" part? 

I get we are trying to say we have looked at it, but the presence of that ignore means we have looked at it and during reviews we are going to have to watch for a) new introductions of ignore and b) any new unsafe calls.

Reply:
@powersj 
- ` -> call audited` removed. 
- `//nolint:gosec` for the whole functions moved to single line filtering